docs/styles/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-size: 16px;
```

Comment:
if you use later `rem` the font-size should be defined on the `html` selector or you can use instead the `:root`-selector